docs/json/sonarr/cf/french-canalplus.json

```diff
@@ -27,7 +27,7 @@
       "negate": false,
       "required": false,
       "fields": {
-        "value": "\\b(C(ANAL)?[ .]?(\\+|Plus))\\b"
+        "value": "\\b(C(?:anal)?[ .]?(?:\\+\\B|Plus))(?:(?<=\\+)\\B|\\b)"
```

Comment:
additional fix, the previous version could not accept `C+` or `Canal.+` because of the trailing `\b`.
`(?:(?<=\+)\B|\b)` states that the regex should end either in a word boundary (if the last character was in `\w`, *or a non word boundary* if the last character seen is a `+`.

It matches with strings such as:

```
Movie 2022 IMAX 2160p Canal plus WEB-DL TrueHD 7.1 Atmos HDR DoVi HEVC-RlsGrp
Movie 2022 IMAX 2160p Canal + WEB-DL TrueHD 7.1 Atmos HDR10 DoVi HEVC-RlsGrp
```

It will not match with strings such as
```
Movie 2022 IMAX 2160p Canal +3 WEB-DL TrueHD 7.1 Atmos DoVi HDR10 HEVC-RlsGrp
Movie 2022 IMAX 2160p Canal Pluss WEB-DL TrueHD 7.1 Atmos DoVi HDR10 HEVC-RlsGrp
```